basic/05_trajectory.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * Generate a trajectory and execute it.
+ *
+ * For more information, go to http://docs.hebi.us/tools.html#cpp-api
+ *
+ * This script assumes you can create a group with 1 module.
+ *
+ * HEBI Robotics
+ * September 2018
+ */
+
+#include "trajectory.hpp"
+#include "Eigen/Eigen"
+
+#include "lookup.hpp"
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "command.hpp"
+#include <math.h>
+#include <chrono>
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
```

Comment:
No

Reply:
If `M_PI` was part of the standard, I'd agree.  Would you prefer ensuring `#define _USE_MATH_DEFINES` is the first line in each example, and that if a user ever includes `<cmath>` or `<math.h>` first, then they will get weird errors?

See, e.g.,: https://stackoverflow.com/questions/26065359/m-pi-flagged-as-undeclared-identifier

Reply:
I remember that. I'd prefer `_USE_MATH_DEFINES` - or even better yet, a simple header under `util` for this purpose.